### PR TITLE
TTS  API improvements

### PR DESCRIPTION
**Pull request by blob42**
**Description**

Continuation of PR #2086

Improvements to the Coqui TTS API/backend.

- [x] #2073: Allow passing speaker_id to models
- [x] Add optional `language` parameter to TTS endpoint/schema
- ~[ ] TTS Info endpoint: List available models, speakers and languages~ (will start new PR for this one)
- [x]  update swagger documentation
- [x] define tts models with config files
- [x] updated docs

**Notes for Reviewers**

The original PR got closed when I force deleted by error the branch while merging

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by blob42** [comment]
Sorry I deleted the branch by error and it closed the PR. If you reopen the previous PR I can force push again there. 

Also there is a lot of diffs due to end of line differences, I work with unix fileformat and it looks like a lot of source code uses DOS (CLRF). Would be great to add a gitattribute to ignore all end of line related diffs

File: core/schema/localai.go
Comment (blob42) [inline core/schema/localai.go:1]: I could not setup git to ignore the end of line space, I am using unix end of line style.

**Comment by blob42** [comment]
@mudler let me know when this PR is planned for review/merge so that I can fix the conflicts at once.

**Review by mudler — APPROVED** [review]
Looks good here, thanks @blob42 - just a matter of getting green lights from CI from my side

**Pull request merged**